__tests__/app.test.js

```diff
@@ -84,6 +84,45 @@ describe("app", () => {
         });
     });
   });
+
+  describe("PATCH - /api/articles/:article_id", () => {
+    test("status: 202 - should accept the request in the form of an object { inc_vote: newVote }", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: 10 })
+        .expect(201)
```

Comment:
Your test description says 202 and you are asserting for a 201...
I would actually say a good old 200 is appropriate here - but a 202 "accepted" not inappropriate either
We would generally reserve 201 for the creation of a brand new resource rather than amendment